operations/customers.md

```diff
@@ -494,6 +495,7 @@ Updates personal information of a customer. Note that if any of the fields is le
 | `DriversLicense` | [Document](customers.md#document) | optional | New drivers license details. |
 | `Address` | [Address](configuration.md#address) | optional | New address details. |
 | `Classifications` | array of [Customer classification](customers.md#customer-classification) | optional | New classifications of the customer. |
+| `Options` | array of [Customer option](customers.md#customer-option) | optional | Options of the customer. |
```

Comment:
Customer option should contain `...` as last item in the list in order to imply that we can add more.